extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.6.json

```diff
@@ -0,0 +1,39 @@
+{
+  "xdm:timestamp": "2020-06-15T10:10:00+00:00",
+  "@id": "https://data.adobe.io/experienceid/5ba0475a-5c31-4fce-b573-8ee222a40f45",
+  "xdm:eventType": "message.feedback",
+  "xdm:identityMap": {
+    "ECID": [
+      {
+        "xdm:id": "68519882713298129995549973016107434638",
+        "xdm:primary": true
+      }
+    ]
+  },
+  "xdm:messageExecution": {
+    "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+    "xdm:messageID": "aa440b96-ce65-4ad2-b95d-8c3b51f89bc2",
+    "xdm:messageLabel": "Deal for Gold members",
+    "xdm:externalApplicationID": "0cc85961-487d-49e5-9b6c-01f5630756ac",
+    "xdm:externalApplication": "Journey"
+  },
+  "xdm:messageProfile": {
+    "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+    "xdm:channel": {
+      "@id": "https://ns.adobe.com/xdm/channels/email"
+    },
+    "xdm:variant": "A"
+  },
+  "xdm:channelContext": {
+    "xdm:pushChannelContext": {
+      "xdm:token": "99156313-c9df-4e54-9c6c-5740f940c3ca",
+      "xdm:platform": "apns",
+      "xdm:appID": "75eafb7e-fa44-4514-86fc-221e32c5aef9"
+    }
+  },
+  "xdm:eventStatus": {
+    "xdm:messageInteraction": {
+      "xdm:interactionType": "action"
```

Comment:
Are there action details missing here?

Reply:
There are no specific requirement provided for the reporting or segmentation use cases on the metadata required with the "action" event.
Though will still check with Pdms for an explicit requirements.
cc: @aashishjassal